src/services/rns/rns.processor.ts

```diff
@@ -66,12 +57,22 @@ async function expirationChangedHandler (logger: Logger, eventData: EventData):
     normalizedTimestamp = eventData.returnValues.expirationTime.slice(5)
   }
   const expirationDate = parseInt(normalizedTimestamp) * 1000
-  await Domain.upsert({ tokenId, expirationDate })
 
-  logger.info(`ExpirationChange event: Domain ${tokenId} updated`)
+  const currentRecord = await DomainExpiration.findOne({ where: { tokenId } })
```

Comment:
Out of curiosity any reason why not to use the `upsert()` here?

Reply:
I just wanted to log when it finds one, which upsert doesnt tell you. But that doesn't matter much as I'll be converting all of this to using the services in the PR that will follow this one.

Reply:
Make sense 👍